docs/frameworks/laravel.md

```diff
@@ -360,3 +360,48 @@ Instead, here is what you need to do:
    ```
 
 All these steps were replacements of running the `passport:install` command [from the Passport documentation](https://laravel.com/docs/passport#installation).
+
+## Lambda Handler Example
+
+If you need to handle custom Lambda events, you can can wire up a handler and respond to events with an Artisan command.
+
+```yml
+# serverless.yml
+
+functions:
+  s3-events:
+    handler: serverless-handler.php
+    timeout: 10 # in seconds
+    events:
+      - s3: ${self:custom.bucket}
+    layers:
+      - ${bref:layer.php-80}
+
+custom:
+  bucket: your-bucket-name
+```
+
+```php
+// serverless-handler.php
+
+<?php declare(strict_types=1);
+
+use Bref\Context\Context;
+use Illuminate\Contracts\Console\Kernel;
+
+require_once __DIR__.'/vendor/autoload.php';
+
+return function ($event, Context $context) {
+    static $app;
+
+    if (null === $app) {
+        $app = require __DIR__.'/bootstrap/app.php';
+    }
+
+    return $app->make(Kernel::class)->call('generate:gif', [
+        'payload' => $event
+    ]);
```

Comment:
Oh, the best practice is not to invoke an artisan command, but to return an actual Lambda handler (which is a PHP class).

Here what you document is a function Lambda handler. What's in the handler is calling an artisan command, but it could do anything. I'm not sure if I'm clear enough, sorry 😕 

https://github.com/brefphp/laravel-bridge/blob/master/worker.php#L18-L21 is an example of that: it doesn't return a function handler, it returns a class handler.

Reply:
Yeap that makes sense. 

Give me a few days to test it, and I'll update the PR.

Thanks for the review!